setup.py

```diff
@@ -179,9 +179,9 @@ def package_files(prefix: str, directory: str, extension: str):
 ]
 
 extra_deps['nlp'] = [
-    'transformers>=4.11,!=4.34.0,<4.42',
+    'transformers>=4.11,!=4.34.0,<4.43',
     'datasets>=2.4,<3',
-    'huggingface-hub>=0.21.2,<0.23',
+    'huggingface-hub>=0.21.2,<0.24',
```

Comment:
just noting, once streaming gets that PR in, people who want to use HF datasets with MDS will use hf 0.23.1 and above. so this is chill